os/drivers/lcd/mipi_lcd.c

```diff
@@ -429,6 +467,7 @@ FAR struct lcd_dev_s *mipi_lcdinitialize(FAR struct mipi_dsi_device *dsi, struct
 		}
 		lcdvdbg("Memory allocated for SW screen rotation, Number of buffers created %d\n", NUM_OF_LCD_BUFFER);
 	}
+	lcd_data_logo = lcd_buffer[0];
```

Comment:
could you let me know what does this mean?

Reply:
There are two buffers, specifically used for rotation purposes.
In bootup time, there are no calls from application, so we can use the rotation buffer.
So, we are using one of the two buffer for logo display purposes.

Reply:
Is there any reason why you set this here even you allocate new memory in the `lcd_set_logo` with `!defined(CONFIG_LCD_SW_ROTATION)`?
It's much difficult to analyse the code.. Let's keep the same location for the same purpose.

Reply:
I have added comment in code and allocated memory in lcd_put_logo function.